### DISCUSSION TITLE
Is it possible to show some HTML for drag-n-dropped element in canvas before rendering actual content?

### DISCUSSION THREAD

**Discussion by DevKishan** [GrapesJS/grapesjs#5304] [Q&A]
I am developing a custom Posts block. This will be used to show WordPress posts. When user drag-n-drop this block in canvas, I need to show Select Format option. After the user select one of the format, the block then should render HTML in that format.

![Peek 2023-08-12 13-58](https://github.com/GrapesJS/grapesjs/assets/21952931/5318371e-5c90-4771-a7b4-71cf3ad0949a)

**Comment by artf** [comment]
It would be a matter of creating a [custom component](https://grapesjs.com/docs/modules/Components.html)
